src/engraving/engravingerrors.h

```diff
@@ -83,7 +83,7 @@ inline Ret make_ret(Err err, const io::path_t& filePath = "")
                .arg(u"https://musescore.org");
         break;
     case Err::FileOld300Format:
-        text = mtrc("engraving", "This file was last saved with a development version of 3.0.");
+        text = mtrc("engraving", "This file was last saved in a development version of 3.0.");
```

Comment:
`using` might be better?

Reply:
I believe this was discussed before, and the decision was to use "in". I'm just making this one consistent with the rest.